backend_addon/{{ cookiecutter.__folder_name }}/Makefile

```
@@ -92,10 +91,34 @@ console: instance/etc/zope.ini ## Start a console into a Plone instance
 create-site: instance/etc/zope.ini ## Create a new site from scratch
 	PYTHONWARNINGS=ignore $(BIN_FOLDER)/zconsole run instance/etc/zope.conf ./scripts/create_site.py
 
-.PHONY: check
-check: $(BIN_FOLDER)/tox ## Check and fix code base according to Plone standards
+# QA
+.PHONY: lint-zpretty
+lint-zpretty: ## Check and fix zcml and xml files
+	@find src -name '*.xml'|xargs uvx zpretty@latest -x --check
+	@find src -name '*.zcml'|xargs uvx zpretty@latest -z --check
```

This should be equivalent:
```suggestion
	uvx zpretty@latest -x --check src --include=\.(xml|zcml)$
```